src/Controllers/Db_auth.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace Cms\Controllers;
+
+use Cms\User\DatabaseUserProvider;
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+
+class Db_auth
+{
+    
+    public $conn;
+    
+    public function userAuth($twig)
+    {
+        $conn = mysqli_connect('localhost', 'root', 'root', 'custom_cms');
```

Comment:
Create a separate ticket for making all these info dynamic.

Reply:
Created the ticket #67 .